lib/content-services/src/lib/dialogs/folder.dialog.scss

```diff
@@ -32,3 +32,14 @@
 .adf-dialog-action-button:disabled > span {
     color: var(--theme-text-color);
 }
+@media screen and (max-width: 380px){
+    .adf-full-width{
+        font-size: 6px;
```

Comment:
this class is used to define the full width (as name says) across many components, why it has to be 6px size?

Reply:
hi @DenysVuika I have changed the adf-full-width class to new class adf-font-size-input

Reply:
why styling only first input and not all of them?

Reply:
noted and resolved